pkg/client/client_v2_test.go

```diff
@@ -284,12 +301,20 @@ func TestClientV2_Checkin_UnitState(t *testing.T) {
 	// receive the units
 	var unitsMu sync.Mutex
 	units := make(map[string]*Unit)
+	var gotFQDN bool
+	var gotTriggers []Trigger
 	go func() {
 		for {
 			select {
 			case <-ctx.Done():
 				return
-			case change := <-client.UnitChanges():
+			case change := <-client.UnitChanged():
+				for _, t := range change.Triggers {
```

Comment:
We should be testing that each of the trigger types works, not just the feature triggers.